.github/workflows/handle_external_pr.yml

```
@@ -0,0 +1,22 @@
+name: Fork Pull Request Workflow
```

I find this wf name a bit ambiguous, how about:
```suggestion
name: Label pull requests from forks
```
or something similar?